src/installer/installer.go

```diff
@@ -286,7 +291,19 @@ func (i *installer) waitForMasterNodes(ctx context.Context, minMasterNodes int,
 	return nil
 }
 
-func (i *installer) updateReadyMasters(nodes *v1.NodeList, readyMasters *[]string) {
+func (i *installer) getInventoryHostsMap(hostsMap map[string]string) map[string]string {
```

Comment:
don't we have same function in controller?

Reply:
in controller it is different, here we don't need try loop

Reply:
why not? sure we will get it in one try?